sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/ErrorResponse.kt

```diff
@@ -0,0 +1,39 @@
+package com.klaviyo.analytics.networking.requests
+
+/**
+ * For parsing Error responses from the backend when an HTTP request fails
+ */
+
+internal data class ErrorResponse(
+    val errors: List<KlaviyoError>
+) {
+    companion object {
+        // Error body constants
+        const val ERRORS = "errors"
+        const val ID = "id"
+        const val STATUS = "status"
+        const val TITLE = "title"
+        const val DETAIL = "detail"
+        const val SOURCE = "source"
+        const val POINTER = "pointer"
+        const val INVALID_INPUT_TITLE = "Invalid input."
```

Comment:
I'm using this to determine if we have an input error vs some other type of 400 error, feels a little shady but if we can guarantee this in every response I think it should be okay. I just don't want to look for an email / phone error on every 400 we get